ddtrace/tracer.py

```diff
@@ -575,10 +543,9 @@ def _check_new_process(self):
 
         if self._runtime_worker is not None:
             self._start_runtime_worker()
-
-        # force an immediate update constant tags since we have reset services
-        # and generated a new runtime id
-        self._update_dogstatsd_constant_tags()
+            # force an immediate update constant tags since we have reset services
+            # and generated a new runtime id
+            self._runtime_worker.update_runtime_tags()
```

Comment:
Does it make sense to do this in `_start_runtime_worker` instead? Seems like we'd always want to update the tags when creating a new runtime worker

Reply:
👍🏽